README.md

```diff
@@ -28,7 +28,7 @@ However, the Datadog Operator is still in beta, so it is not yet a recommended w
 
 ## Getting started
 
-See the [Getting Started][5] dedicated documentation to learn how to deploy the Datadog operator and your first Agent, and [Configuration][12] to see examples and a list of all configuration keys.
+See the [Getting Started][5] dedicated documentation to learn how to deploy the Datadog operator and your first Agent, and Configuration ([v1alpha1][12] or [v2alpha1][13]) to see examples and a list of all configuration keys.
```

Comment:
`v2alpha1` is not ready for public use yet - until then, can we leave that off of this doc?

Reply:
Ah, good to know. Could I make a suggestion then that we put a warning on top of the `v2alpha1` markdown page saying something like "WIP, not ready for public use"?

'Cause when I poked around in the repo, I assumed that was ready for use.

Reply:
yes definitely, we can do that - thanks for pointing it out